routes/user.js

```diff
@@ -0,0 +1,113 @@
+const express = require("express");
+const router = express.Router();
+const gravatar = require("gravatar");
+const multer = require("multer");
+const sharp = require("sharp");
+const fs = require("fs");
+const User = require("../models/user");
+const authMiddleware = require("../middleware/authMiddleware");
+
+const upload = multer({ dest: "tmp/" });
+const bcrypt = require("bcryptjs");
+const { body, validationResult } = require("express-validator");
```

Comment:
Zamiast tego joi powinno być. A jesli juz sie na to decydujesz to usuń joi. Albo jedno albo drugie.